htdocs/sites/obhistory.php

```
@@ -90,16 +108,36 @@ function sky_formatter($row)
     return sprintf("%.0f", $val);
 }
+
+/**
+ * Format visibility values for display, returning an empty string for null values
+ * @param float|null $val visibility value in miles
+ * @return string formatted visibility value or empty string if null
+ */
 function vis_formatter($val)
 {
     if (is_null($val)) return "";
     return round($val, 2);
```

PHPStan (2.2.2)

[error] Function vis_formatter() should return string but returns float.


(return.type)